https://f3d.app/web not work

**Describe the bug**
I downloaded the [example files](https://drive.google.com/file/d/1hYGAHr67T7tBrUKFrCF_uc_jnvJaQF-A/view?usp=sharing)
Tested the linked viewer https://f3d.app/web/

**To Reproduce**
Steps to reproduce the behavior:
 1. Open the website https://f3d.app/web/
 2. Open
 3. Choose `DamagedHelmet.glb` or any other supported files
 4. nothing happens
 5. press F12
```
Loading failed for the <script> with the source "https://f3d.app/web/f3d.js". web:87:47
Uncaught ReferenceError: f3d is not defined
    <anonymous> https://f3d.app/web/:117
```


**Expected behavior**
See the files 

**System Information:**
 - OS: Ubuntu 24.04 
 - GPU and GPU driver: VM-Ware
 - Browser: Firefox

Here's the error: https://github.com/f3d-app/f3d/actions/runs/11271374639/job/31344070485#step:3:235  
```
/src/vtkext/private/module/vtkF3DGenericImporter.cxx:163:9: error: no member named 'ActorCollection' in 'vtkF3DGenericImporter'
  163 |   this->ActorCollection->AddItem(this->Pimpl->GeometryActor);
      |   ~~~~  ^
1 error generated.
```

I think the wasm Docker image needs a VTK update. The CI job is passing even if there is a compilation error, which should not happen.

--- a/webassembly/F3DEmscriptenBindings.cxx
+++ b/webassembly/F3DEmscriptenBindings.cxx
@@ -100,7 +100,7 @@ EMSCRIPTEN_BINDINGS(f3d)
 
   // f3d::scene
   emscripten::class_<f3d::scene>("Scene")
-    .function("supported", &supported, emscripten::allow_raw_pointers())
+    .function("supports", &supports, emscripten::allow_raw_pointers())
     .function("add", &add, emscripten::allow_raw_pointers())
     .function("clear", &clear, emscripten::allow_raw_pointers());
 

--- a/webassembly/build.sh
+++ b/webassembly/build.sh
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -euo pipefail
+
 cmake -S /src -B /build \
     -DBUILD_SHARED_LIBS=OFF \
     -DF3D_STRICT_BUILD=ON \

